Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/DiagnosticScreenshotBuilder.cs

```diff
@@ -43,8 +45,12 @@ public static DiagnosticScreenshotBuilder CreateBrowserScreenshot ([NotNull] IBr
       ArgumentUtility.CheckNotNull("contentLocator", contentLocator);
       ArgumentUtility.CheckNotNull("browserSession", browserSession);
 
+      var screenshot = Screenshot.TakeBrowserScreenshot(browserSession, contentLocator);
+      if (browserSession.Headless)
+        contentLocator = new FixedBrowserContentLocator(new Rectangle(0, 0, screenshot.Image.Width, screenshot.Image.Height));
```

Comment:
should we find a way to make this thing transparent (future improvment)?

Reply:
What do you mean with "transparent"?

Reply:
right now, you need to post-process the result of Screenshot.TakeBrowserScreenshot manually inside your tests isntead of getting this done automatically. This seems like a possible cause for problems and needless complexity on the user-side.

Reply:
After looking at it again I have concluded that the locator that is passed in is irrelevant. Since the screenshot is in the `Browser` coordinate system the locator is never needed. Thus, we have a few options:

 1. Remove `FixedBrowserContentLocator` and pass the provided `contentLocator`
 2. Always pass a `FixedBrowserContentLocator`
 3. Pass a new locator that always throws to make sure that nobody tries to call the locator.

I prefer option 1 as it keeps things simple

Reply:
Option 1 sounds good.